### ISSUE TITLE
Priming work_dir

### ISSUE THREAD

**Issue by yrashk** [tcdi/plrust#324]
When using plrust for the first time, it takes ages to compile the first function, which is not an ideal user experience.

It does, however, help to create a function once before using the extension to prime it. Here's an [example of how priming can be done](https://github.com/omnigres/omnigres/blob/master/Dockerfile#L114-L118).

This can apply to .deb building, .deb post-install, or other extensions' lifecycle points. Also, the upgrade story should be considered as well.

**Comment by eeeebbbbrrrr** [comment]
yeah, we'll take a look at this, probably next week, with an aim to get such a thing included in our .debs for the next release.  Thanks!

**Comment by eeeebbbbrrrr** [comment]
We've spent quite a bit of time discussing this internally and the consensus here is that it'll likely be too error-prone for PL/Rust's .deb (and future other package types) packaging to include something that builds an initial function to "prime" the `target/` directory.

Our goal, such as we can accomplish it, is to continue to improve overall compilation speed.  We've got a number of ideas around this that we haven't yet explored.
